### ISSUE TITLE
The training is unstable

### ISSUE THREAD

**Issue by shengkelong** [pkuxmq/Invertible-Image-Rescaling#35]
Thank you for your impressive work. But when I try to recurrent this network(I rewrite the code myself), sometimes the loss will suddenly increase by 10 times. The structure of the network is correct because I can load the pretrained network, so I think there may be some details I didn't notice. Could you tell me what methods you have taken in training to ensure stability?

**Comment by pkuxmq** [comment]
We restrict the range on exp() and apply gradient clipping. For reasons please refer to https://github.com/pkuxmq/Invertible-Image-Rescaling/issues/24.

**Comment by Feynman1999** [comment]
> Thank you for your impressive work. But when I try to recurrent this network(I rewrite the code myself), sometimes the loss will suddenly increase by 10 times. The structure of the network is correct because I can load the pretrained network, so I think there may be some details I didn't notice. Could you tell me what methods you have taken in training to ensure stability?

I have also encountered this issue recently, despite using gradient clipping strategy. How did you solve it?
